<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageMore</class>
 <widget class="QWidget" name="PageMore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>532</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>More</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #FFFFFF</string>
  </property>
  <widget class="QFrame" name="headerFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #FCD535;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #474D57;</string>
    </property>
    <property name="text">
     <string>More</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="moreVerticalFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>311</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <layout class="QVBoxLayout" name="moreVerticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>12</number>
    </property>
    <item>
     <widget class="QComboBox" name="accountsComboBox">
      <property name="minimumSize">
       <size>
        <width>119</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>true</bold>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
    border: 1px solid lightgray;
    padding: 1px 18px 1px 3px;
    min-width: 6em; 
	color:black;
}
QComboBox:on { /* shift the text when the popup opens */
    padding-top: 3px;
    padding-left: 4px;
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;

    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 5px; /* same radius as the QComboBox */
    border-bottom-right-radius: 5px;
}
QComboBox::drop-down::hover {
	background-color: rgba(0, 0, 0, 16);
}
QComboBox::drop-down::pressed {
    background-color: rgba(0, 0, 0, 64);
}
QComboBox::down-arrow {
    image: url(:/res/ic/res/ic/dropDownArrow.png);
	width: 12px;
    height: 12px;
}
QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}
QListView::item {
	height: 31px;
	background-color: white;
	color: black;
}
QListView::item:selected:!active, QListView::item:selected:active {
	background-color: lightgrey;
	color: black;
}
QListView::item::hover {
	background-color: grey;
	/*font: url(:/Resources/Fonts/GameJam.ttf);*/
	color: black;
}
</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="newAccountPushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>true</bold>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-style: solid;
	background-color: rgba(0, 0, 0, 8);
	border-width: 1px;
	border-color: lightgrey;
	text-align:left;
    padding-right: 3px;
    padding-left: 3px;
	color:black;
}
QPushButton:hover {
    background-color: rgba(0, 0, 0, 16);
}

QPushButton:pressed {
    background-color: rgba(0, 0, 0, 64);
}</string>
      </property>
      <property name="text">
       <string>NEW ACCOUNT</string>
      </property>
      <property name="icon">
       <iconset resource="../../res.qrc">
        <normaloff>:/res/ic/res/ic/newAccount.png</normaloff>:/res/ic/res/ic/newAccount.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="explorerPushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>true</bold>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-style: solid;
	background-color: rgba(0, 0, 0, 8);
	border-width: 1px;
	border-color: lightgrey;
	text-align:left;
    padding-right: 3px;
    padding-left: 3px;
	color:black;
}
QPushButton:hover {
    background-color: rgba(0, 0, 0, 16);
}

QPushButton:pressed {
    background-color: rgba(0, 0, 0, 64);
}</string>
      </property>
      <property name="text">
       <string>EXPLORER</string>
      </property>
      <property name="icon">
       <iconset resource="../../res.qrc">
        <normaloff>:/res/ic/res/ic/magnify.png</normaloff>:/res/ic/res/ic/magnify.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="settingsPushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>true</bold>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-style: solid;
	background-color: rgba(0, 0, 0, 8);
	border-width: 1px;
	border-color: lightgrey;
	text-align:left;
    padding-right: 3px;
    padding-left: 3px;
	color:black;
}
QPushButton:hover {
    background-color: rgba(0, 0, 0, 16);
}

QPushButton:pressed {
    background-color: rgba(0, 0, 0, 64);
}</string>
      </property>
      <property name="text">
       <string>SETTINGS</string>
      </property>
      <property name="icon">
       <iconset resource="../../res.qrc">
        <normaloff>:/res/ic/res/ic/settings.png</normaloff>:/res/ic/res/ic/settings.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="aboutUsPushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>true</bold>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-style: solid;
	background-color: rgba(0, 0, 0, 8);
	border-width: 1px;
	border-color: lightgrey;
	text-align:left;
    padding-right: 3px;
    padding-left: 3px;
	color:black;
}
QPushButton:hover {
    background-color: rgba(0, 0, 0, 16);
}

QPushButton:pressed {
    background-color: rgba(0, 0, 0, 64);
}</string>
      </property>
      <property name="text">
       <string>ABOUT US</string>
      </property>
      <property name="icon">
       <iconset resource="../../res.qrc">
        <normaloff>:/res/ic/res/ic/aboutUs.png</normaloff>:/res/ic/res/ic/aboutUs.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="advancedFunctionsPushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>true</bold>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-style: solid;
	background-color: rgba(0, 0, 0, 8);
	border-width: 1px;
	border-color: lightgrey;
	text-align:left;
    padding-right: 3px;
    padding-left: 3px;
	color:black;
}
QPushButton:hover {
    background-color: rgba(0, 0, 0, 16);
}

QPushButton:pressed {
    background-color: rgba(0, 0, 0, 64);
}</string>
      </property>
      <property name="text">
       <string>ADVANCED FUNCTIONS</string>
      </property>
      <property name="icon">
       <iconset resource="../../res.qrc">
        <normaloff>:/res/ic/res/ic/aboutUs.png</normaloff>:/res/ic/res/ic/aboutUs.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../res.qrc"/>
 </resources>
 <connections/>
</ui>
